CHANGELOG.md

```diff
@@ -9,13 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- [1640](https://github.com/nf-core/sarek/pull/1620) - Add `lofreq` as a tumor-only variant caller.
+- [1640](https://github.com/nf-core/sarek/pull/1620) - Add `lofreq` as a tumor-only variant caller
 - [1642](https://github.com/nf-core/sarek/pull/1642) - Back to dev
 
 ### Changed
 
 ### Fixed
 
+- [1656](https://github.com/nf-core/sarek/pull/1656) - Retiring parameter `snpeff_genome`
```

Comment:
Can you add a deprecation in the Parameters section, and also specify that the snpeff_db parameter usage has been updated?

Reply:
nevermind, I need this for https://github.com/nf-core/sarek/pull/1663, so I'll merge it and update the CHANGELOG